documentation/library_guide/overview.rst

```diff
@@ -83,6 +83,11 @@ Known Limitations
   To overcome these issues, include oneDPL header files before the standard C++ header files,
   or disable parallel algorithms support in the standard library. 
   For more information, please see `Intel® oneAPI Threading Building Blocks (oneTBB) Release Notes`_.
+* When using oneDPL with serial or OpenMP backend and libstdc++ version 9, and C++17 or C++20, a compilation error occurs
```

Comment:
I have made the following edits, please update and this can be closed. 

- A compilation error occurs if oneTBB is not in your environment when you use oneDPL with a serial or OpenMP backend, libstdc++ version 9, and C++17 or C++20. The error occurs because libstdc++ version 9 does not check for oneTBB availability. To remove the error:
  -  Disable support for your Parallel STL algorithms by setting the macro, ``PSTL_USE_PARALLEL_POLICIES`,` to zero before including the first standard header file in each translation unit. 
  -  Include oneDPL headers before you include other headers.

Reply:
Good suggestion, thanks.
A small correction, I believe that the word "your" in this line is superfluous:

> Disable support for your Parallel STL algorithms by setting the macro, ``PSTL_USE_PARALLEL_POLICIES, to zero before including the first standard header file in each translation unit.

Reply:
I am fine with removing "your".

Reply:
Thanks for review, applied your suggestions.